frontend/src/pages/blocks/Blocks.tsx

```diff
@@ -3,27 +3,38 @@ import { useSearch } from '../../shared/hooks/search';
 import { useTimeoutPolling } from '../../shared/hooks/timeout-polling';
 
 import classes from './Blocks.module.scss';
+import { useFilterData } from '../../shared/hooks/filter-data';
 
 const Blocks = () => {
     const { searchTerm, setPlaceholder } = useSearch();
     const { data } = useTimeoutPolling('/api/blocks/polling');
+    const { filteredData } = useFilterData(data, searchTerm);
 
     useEffect(() => {
         setPlaceholder('Search blocks');
     }, []);
 
+    useEffect(() => {
+        console.log('filtered data changed', filteredData);
+    }, [filteredData]);
+
     return (
         <div>
             <h2>Blocks</h2>
             <p>Search term: {searchTerm}</p>
             {data && <p>Data length: {data?.length}</p>}
+            {filteredData && <p>Filtered data length: {filteredData?.length}</p>}
 
             <div className={classes.listContainer}>
-                {data &&
-                    data.length &&
-                    data.map((d: any, index: number) => (
+                {filteredData &&
+                    filteredData.length &&
```

Comment:
As I understand, `filteredData` can be only empty or non-empty array, so there is no need to check that it's "truthty" and "has length" ? Should we just map over items like: `<div> { filteredData.map(e => ... ) }  </div>` ?

Reply:
this part is was only done to test other functionality = crap code :)